### PR TITLE
feat: add store for language and stick mlanguage modal to header

### PR DESCRIPTION
**Pull request by MihailMatveichuk**
- Add store for language
- Stick modal of language for header
- Refactor styles into CSS modules
- change logo and add language block into qustioners header
Screenshot_1
Screenshot_2
Screenshot_3

### CODE REVIEW

File: src/components/Header/LanguageSelect/LanguageSelect.module.css [hunk 1]
```diff
@@ -1,58 +1,39 @@
...
   background: #32303e;
-  position: absolute;
-  top: 40px;
-  left: -50px;
-  z-index: 1;
-  display: block;
+
   opacity: 1;
-  animation: languagevisible 0.5s ease-in-out;
+  animation: languagevIsVisible 0.5s ease-in-out;
```
Comment (akrutsko) [inline src/components/Header/LanguageSelect/LanguageSelect.module.css:8]:
```suggestion
  animation: languagevIsVisible 0.5s ease-in-out;
  animation: languageIsVisible 0.5s ease-in-out;
```

File: src/components/Header/LanguageSelect/LanguageSelect.module.css [hunk 2]
```diff
@@ -61,9 +42,9 @@
   font-style: normal;
   font-weight: 400;
   line-height: 14px;
-  letter-spacing: 0.1px;
 }
-@keyframes languagevisible {
+
+@keyframes languagevIsVisible {
```
Comment (akrutsko) [inline src/components/Header/LanguageSelect/LanguageSelect.module.css:47]:
```suggestion
@keyframes languageIsVisible {
```

File: src/components/Header/LanguageSelect/LanguageSelect.tsx [hunk 3]
```diff
@@ -1,46 +1,45 @@
...
-import { ReactComponent as ActiveArrow } from '../../../assets/Header/activeArrow.svg';
-import { ReactComponent as PassiveArrow } from '../../../assets/Header/passiveArrow.svg';
+import { setLanguage } from 'store/language/slice';
+import { useAppDispatch } from 'hooks/hooks';
+import styles from './LanguageSelect.module.css';
 
-export const LanguageSelect = () => {
-  const [activeMenu, setActiveMenu] = useState(false);
-  const [selectLang, setselectLang] = useState('English');
+// import '../LanguagesSwich.css';
```
Comment (akrutsko) [inline src/components/Header/LanguageSelect/LanguageSelect.tsx:8]: commented code

File: src/components/Header/LanguageSelect/LanguageSelect.tsx [hunk 4]
```diff
@@ -1,46 +1,45 @@
...
 
-  const handleActiveMenu = () => {
-    setActiveMenu(!activeMenu);
-  };
+export const LanguageSelect = ({
+  isActiveMenu,
+  setIsActiveMenu,
+}: {
+  isActiveMenu: boolean;
+  setIsActiveMenu: React.Dispatch<React.SetStateAction<boolean>>;
```
Comment (akrutsko) [inline src/components/Header/LanguageSelect/LanguageSelect.tsx:15]:
```suggestion
  setIsActiveMenu: Dispatch<SetStateAction<boolean>>;
```

File: src/components/Header/LanguageSelect/LanguageSelect.tsx [hunk 5]
```diff
@@ -1,46 +1,45 @@
...
+}) => {
+  const dispatch = useAppDispatch();
   const switchLang = (event: React.MouseEvent) => {
```
Comment (akrutsko) [inline src/components/Header/LanguageSelect/LanguageSelect.tsx:18]:
```suggestion
  const switchLang = (event: MouseEvent) => {
```

File: src/components/Layouts/MainLayout/MainLayout_Header/DropdownContent/DropdownContent.tsx [hunk 6]
```diff
@@ -16,17 +19,21 @@ type MenuItem = {
 type DropdownContentProps = {
   closeDropdown: () => void;
   setModalOpen: (isOpen: boolean) => void;
+  setLanguageModalVisible: React.Dispatch<React.SetStateAction<boolean>>;
```
Comment (akrutsko) [inline src/components/Layouts/MainLayout/MainLayout_Header/DropdownContent/DropdownContent.tsx:22]:
```suggestion
  setLanguageModalVisible: Dispatch<SetStateAction<boolean>>;
```

File: src/components/Layouts/MainLayout/MainLayout_Header/DropdownContent/DropdownContent.tsx [hunk 7]
```diff
@@ -16,17 +19,21 @@ type MenuItem = {
...
+  setLanguageModalVisible,
 }) => {
   const openModalPremium = () => {
     setModalOpen(true);
     closeDropdown();
   };
 
+  const languageName = useAppSelector(getLanguage);
+
   const handleLinkClick = (e: React.MouseEvent<HTMLDivElement>) => {
```
Comment (akrutsko) [inline src/components/Layouts/MainLayout/MainLayout_Header/DropdownContent/DropdownContent.tsx:37]:
```suggestion
  const handleLinkClick = (e: MouseEvent<HTMLDivElement>) => {
```

File: src/components/Layouts/MainLayout/MainLayout_Header/mainLayoutHeader.module.css [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+.notifications {
+  position: relative;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.count_notifications {
```
Comment (akrutsko) [inline src/components/Layouts/MainLayout/MainLayout_Header/mainLayoutHeader.module.css:87]: countNotifications

File: src/components/ModalNotification/ModalNotification.tsx [hunk 9]
```diff
@@ -12,6 +12,8 @@ type NotificationProp = {
   onClose?: () => void;
 };
 
+const maxHeight = '600px';
```
Comment (akrutsko) [inline src/components/ModalNotification/ModalNotification.tsx:15]:
```suggestion
const MAX_HEIGHT = '600px';
```

**Comment by MihailMatveichuk** [comment]
Thanks for commiting !!

**Pull request merged**